.github/workflows/e2e.yml

```diff
@@ -21,56 +22,53 @@ env:
 jobs:
   e2e:
     concurrency:
-      group: kogito-serverless-operator-e2e-tests-${{ matrix.cluster-type }}-${{ github.head_ref }}
+      group: kogito-serverless-operator-e2e-tests-${{ github.head_ref }}
       cancel-in-progress: true
     timeout-minutes: 120
-    strategy:
-      matrix:
-        cluster-type:
-        - kind
-        - minikube
-      fail-fast: false
     runs-on: ubuntu-latest
-    name: End-to-end tests (${{ matrix.cluster-type }})
+    name: End-to-end tests (Minikube)
     steps:
-    - name: Checkout Code
-      uses: actions/checkout@v3
-    
-    - name: Install Go
-      id: go
-      uses: actions/setup-go@v3
-      with:
-        go-version: ${{ env.GO_VERSION }}
-        cache: true
+      - name: Checkout Code
+        uses: actions/checkout@v3
 
-    - name: Setup Minikube cluster
-      if: ${{ matrix.cluster-type == 'minikube' }}
-      id: minikube
-      uses: medyagh/setup-minikube@latest
-      with:
-        addons: registry,metrics-server
-        kubernetes-version: ${{ env.KUBERNETES_VERSION }}
-        # https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources
-        cpu: max
-        memory: max
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GO_VERSION }}
+          cache: true
 
-    - name: Setup KinD cluster
-      if: ${{ matrix.cluster-type == 'kind' }}
-      id: kind
-      uses: container-tools/kind-action@v2
-      with:
-        version: v0.17.0
-        node_image: kindest/node:${{ env.KUBERNETES_VERSION }}
-        kubectl_version: ${{ env.KUBERNETES_VERSION }}
-        wait: 120s
+      - name: Build operator image
+        run: |
+          make container-build BUILDER=docker IMG=${{ env.OPERATOR_IMAGE_NAME }}
```

Comment:
can't we switch to podman?

Reply:
well I need to make sure it works. let's try !